mephisto/client/review/review_server.py

```diff
@@ -85,10 +89,67 @@ def consume_data():
             ready_for_next.wait()
         finished = True
 
+    def consume_all_data(page, results_per_page):
+        limit = (
+            results_per_page
+            if (
+                results_per_page is not None
+                and isinstance(results_per_page, int)
+                and results_per_page > 0
+            )
+            else RESULTS_PER_PAGE_DEFAULT
+        )
+        paginated = (
+            True if page is not None and isinstance(page, int) and page > 0 else False
```

Comment:
you don't need to do a None check if you're also doing isinstance, you can just do the isinstance, since it will also cover the None clause (a value of None will always make isinstance false, i.e. `isinstance(None, int) == false`)

that is to say you could just shorted all of the above with:
```python
paginated = type(page) is int and page > 0
```